### ISSUE TITLE
Create seed file

### ISSUE THREAD

**Issue by LucasAndrad** [LearnITGirl/WebPlatform#155]
Is it possible for some one who understand all the code and all the classes, create a simple seed file? Because with all this models like "MenteeApplicationEvaluation", "MenteeApplicationValidation", "MentorApplicationEvaluation" etc.. It's hard to understand every relations and populate the database from zero. So a simple and basic seed file will help new contributors to use and understand the application through localhost.

**Comment by RMaksymczuk** [comment]
Sure, I can prepare a seed file with some test data to load on localhost and I'll try to cleanup the production one. Also, it might be a good idea, to write down all the steps of application and evaluation processes, together with the models that are used. I'll let you know as soon as I finish.

**Comment by LucasAndrad** [comment]
Thanks!!! This will help a lot.

**Comment by LucasAndrad** [comment]
Hi @RMaksymczuk, any status for this issue?

**Comment by RMaksymczuk** [comment]
Hi, sorry, I had no time last week to work on that, but I plan to do it in next 2-3 days :)

**Comment by LucasAndrad** [comment]
Hey @RMaksymczuk, sorry to bother you again, really. Do you have any status of seed file?

**Comment by RMaksymczuk** [comment]
No problem, in fact I needed this motivational boost to find some time to deliver that in some reasonable state. Unfortunately it isn't so easy as we never focused on organizers/admin section and through the time, the whole project flow massed up a bit :)

The information I included are not ideal. I've started working on the /organizers section as it definitely need a refactoring and clean up. I'll get back with another pull request and contribution updates when it'll be finished. 

Till then, let me know if the information I provided helped you. If you have any specific questions or concerns about some parts of code, I can explain it more extensively. Maybe this way we'll be able to expand a contribution information.
